specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimbackends.json

```diff
@@ -632,6 +683,33 @@
         }
       ],
       "description": "Parameters supplied to the Update Backend operation."
+    },
+    "BackendReconnectContract": {
+      "properties": {
+        "properties":{
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/BackendReconnectProperties",
+          "description": "Reconnect request properties."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./apimanagement.json#/definitions/Resource"
+        }
+      ],
+      "description": "Reconnect request parameters."
+    },
+    "BackendReconnectProperties": {
+      "properties": {
+        "after": {
```

Comment:
Is there a chance for this property name to be updated? "after" seems to generic. May be something like reconnectAfter or similar?

Reply:
@ravbhatnagar I'm afraid we can't rename the property. But, as the property is within 'reconnect contract' scope, it should be pretty much self-explanatory to customers (customer will be doing `backends/id/reconnect`  and then specify `after` in the body).

Reply:
@promoisha - Does this mean this has already been deployed to the service? I would request you team to get API reviews done before code is deployed on the service side. It defeats the purpose of API reviews otherwise. Please let me know if I need to communicate this more broadly in the team.
Yes context is there but even then propertyNames should be self explanatory as much as possible.

Reply:
@ravbhatnagar Yes, it has already been deployed. We'll discuss the possibility of getting API reviews done before deployments.

Reply:
@promoisha - This should not be a possibility but a requirement, specially for major API changes and new APIs. If there are concerns, please let me know and I can communicate this to Taylor/Kedar.

Reply:
@ravbhatnagar it'd be great if you could reach out to Taylor or/and Kedar regarding the review process for future changes.